src/components/Button/CartControlButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { AddCartStyle, RemoveCartStyle } from "./Button.style";
+import AddCart from "../../assets/add-cart.svg";
+import RemoveCart from "../../assets/remove-cart.svg";
+
+interface CartControlButtonProps {
+  isInCart: boolean;
+  onClick: () => void;
+}
+
+const CartControlButton = ({ isInCart, onClick }: CartControlButtonProps) => {
+  return (
+    <>
+      {isInCart ? (
+        <RemoveCartStyle onClick={onClick}>
+          <img src={RemoveCart} alt="장바구니 삭제" />
+          <span>빼기</span>
+        </RemoveCartStyle>
+      ) : (
+        <AddCartStyle onClick={onClick}>
+          <img src={AddCart} alt="장바구니 추가" />
+          <span>담기</span>
+        </AddCartStyle>
+      )}
+    </>
+  );
```

Comment:
JSX문 전체가 분기되는 경우 아래처럼도 적어볼 수 있을거 같아요
```suggestion
  if (isInCart) {
    return (
        <RemoveCartStyle onClick={onClick}>
          <img src={RemoveCart} alt="장바구니 삭제" />
          <span>빼기</span>
        </RemoveCartStyle>    
    )
  }
  return (
        <AddCartStyle onClick={onClick}>
          <img src={AddCart} alt="장바구니 추가" />
          <span>담기</span>
        </AddCartStyle>
  );
```

Reply:
(변경하라는 얘기는 아닙니다)

Reply:
아하!
꼭 변경 하라는 건 아니라고 하셨는데, 혹시 제안해주신 방향이 어떤 점 때문에 추천해주신 걸까요? 가독성 측면에서 더 좋은가요? 🤔

Reply:
개인적으로는 가독성 측면에서 좋다고 생각해서 말씀드렸습니다 ㅎ ㅎ 이 부분은 취향이 갈릴 수도 있어서 참고만 하시면 좋을거 같아요